docs/atlascli/command/atlas-auth-login.txt

```diff
@@ -44,10 +44,6 @@ Options
      - 
      - false
      - Don't try to open a browser session.
-   * - --skipConfig
```

Comment:
We are removing this option entirely? So users without a profile can no longer skip profile configuration if they want to?

Reply:
based on the [conversation](https://jira.mongodb.org/browse/CLOUDP-116868?focusedCommentId=4537840&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4537840) we had (Jakub, Bianca and Gustavo), for now, we're only deprecating the flag.
The flag won't be included anymore in the docs, but if there are still users that are using it, they will get this warning: `Flag --skipConfig has been deprecated, if profile is configured, login flow skips by default the config step.` (also part of this PR)

Reply:
Ok, thanks for the context. This will have impact on the manually written docs as well so I will open a DOCSP ticket for the change.